src/diff_tform.c

```diff
@@ -296,6 +296,9 @@ static int normalize_find_opts(
 	if (opts->flags & GIT_DIFF_BREAK_REWRITES)
 		opts->flags |= GIT_DIFF_FIND_REWRITES;
 
+	if (opts->flags & GIT_DIFF_FIND_NO_RENAMES)
+		opts->flags &= ~GIT_DIFF_FIND_ALL;
```

Comment:
This is pretty weird.  You end up turning off your own bit, after doing some unnecessary config lookups.  Why not just exit this function early if `GIT_DIFF_FIND_NO_RENAMES` is set?

Also, `GIT_DIFF_FIND_NO_RENAMES` seems like an odd name (since it also turns off copy detection)?  Does `GIT_DIFF_FIND_NONE` make more sense?